dbt/models/default/schema.yml

```diff
@@ -122,13 +258,29 @@ models:
   - name: default.vw_pin_land
     description: '{{ doc("vw_pin_land") }}'
   - name: default.vw_pin_sale
-    description: '{{ doc("vw_pin_sale") }}'
```

Comment:
This isn't actually getting deleted, it's just being moved below the `columns` definition.